lib/from_erlang.ml

```diff
@@ -13,8 +13,16 @@ let const_of_literal = function
   | LitInteger (_line_t, i) -> Number i
   | LitBigInt (_line_t, z) ->
      raise Known_error.(FialyzerError (NotImplemented {issue_links=["https://github.com/dwango/fialyzer/issues/93"]; message="support bigint literal"}))
-  | LitString (_line_t, s) ->
-     raise Known_error.(FialyzerError (NotImplemented {issue_links=["https://github.com/dwango/fialyzer/issues/90"]; message="support string(list) literal"}))
+  | LitString (_line_t, _s) ->
+     (* treat in expr_of_literal, etc *)
+     failwith "cannot reach here"
```

Comment:
When https://github.com/dwango/fialyzer/pull/119 is merged, I have to add `pattern_of_literal` function.

Reply:
done